content/hub/template_http_js.md

```diff
@@ -0,0 +1,22 @@
+title = "JavaScript HTTP trigger template"
+template = "render_hub_content_body"
+date = "2022-10-15T00:22:56Z"
+content-type = "text/html"
+tags = ["javascript", "http"]
+
+[extra]
+author = "fermyon"
+type = "hub_document"
+category = "Template"
+language = "JavaScript"
```

Comment:
```suggestion
language = "JavaScript"
```

I am still on the fence about if we want to differentiate between JS and TS though. Do not have strong opinions here but I am looking for some thoughts on this.

Reply:
I do not have strong opinions either (I know, is this my kidnap message) - this was based on the templates having two different URLs - but we could still categorise them under one language of course